bundles/org.openhab.binding.tellstick/pom.xml

```diff
@@ -15,7 +15,7 @@
   <name>openHAB Add-ons :: Bundles :: Tellstick Binding</name>
 
   <properties>
-    <bnd.importpackage>!com.luckycatlabs.*,!com.jcraft.jzlib.*,!org.apache.commons.cli.*,!org.eclipse.swt.*</bnd.importpackage>
+    <bnd.importpackage>!com.luckycatlabs.*,!com.jcraft.jzlib.*,!org.apache.commons.cli.*,!org.eclipse.swt.*,!javax.activation</bnd.importpackage>
```

Comment:
This is not needed if I compile locally?

Reply:
No it is not needed when compiling but during runtime.

Reply:
If it's needed during runtime you must not exclude it. Instead you need to add the required dependency. But I believe that is already present in the target platform anyway, otherwise the karaf plugin should fail to resolve the feature.

Reply:
No I meant that the _change_ is not needed to be able to compile but it is there to sort out a runtime issue. So in fact it is the opposite, the new version of async-http-client has javax.activation as a transitive dependency but this addon does not seem to use those parts of the async-http-client library so that is why I excluded javax.activation rather than adding the dependency for it.

Reply:
Maybe. Fine if it works. The other option would be to add

```
mvn:org.apache.servicemix.specs/org.apache.servicemix.specs.activation-api-1.2.1/1.2.1_2
```

to the feature. That is activation API we use in other bundles. But fine with me if it works with the exclusion. Up to you.

Reply:
Yea maybe that is a nicer solution even though it is not used.
I added that bundle as well and removed the exclude.